simularium/simularium-website
# Fix/share download styling

Problem
=======
@lynwilhelm pointed out some styling mistakes on the`Share` and`Download` buttons, and added a couple tweaks not in the original designs.
Designs [here](https://www.figma.com/file/9i8Dphcfp3t7e7TouwNheB/Simularium-share-time?type=design&node-id=96-1941&mode=design&t=Of50s0v0xSpiAGTZ-0) and [here](https://xd.adobe.com/view/75b6a3c8-f650-443b-bccc-d1fcd134a35e-20e5/screen/d97dd2b2-08b4-4fe1-ae09-1dddb764240b)

This PR should fix those problems and I added a couple readability changes. I went over the visual changes with @lynwilhelm in a huddle so feel confident this branch reflects design intentions.

Styling changes:
- remove pipe divider
- swap button order
- icons slightly larger
- active button hover state should be lighter color
- tooltips should be closer to buttons and more centered
- new tooltip text

Note about selectors: `.container :global(.ant-tooltip-open.ant-btn)`
The hover state and tooltip open behavior were not in sync, so this rule is used of the `:hover` pseudo-class.

@lynwilhelm  also noticed a bug that I could not reproduce where a tooltip would open automatically when a trajectory is loaded, if the user had hovered over the share button BEFORE loading the trajectory. I'm crossing my fingers that some of these changes will resolve that issue, but please let me know if you can reproduce it on this branch.

* Bug fix (non-breaking change which fixes an issue)

Steps to Verify:
----------------
1. Run website
2. Check button styling before and after loading trajectory
3. From main page: hover over share button, then load trajectory
4. See if tooltip opens erroneously

Screenshots:
----------------

Before, disabled:
<img width="410" alt="incorrectdisabled" src="https://github.com/simularium/simularium-website/assets/24981838/9cbb6826-97a7-46c6-a033-1132453bdf70">

Before, hover:
<img width="406" alt="incorrecthover" src="https://github.com/simularium/simularium-website/assets/24981838/502e7580-9fab-4b05-9438-a61d5b24dee2">

<img width="392" alt="incorrectdownload" src="https://github.com/simularium/simularium-website/assets/24981838/ea748536-b541-45d3-8f7a-21d1f43f7bdd">


After, disabled:
<img width="542" alt="correctdisabled" src="https://github.com/simularium/simularium-website/assets/24981838/1ff0896b-e3d1-4e07-835b-ea73c6985d1b">

After, hover:
<img width="533" alt="correcthover" src="https://github.com/simularium/simularium-website/assets/24981838/0a101020-fbe9-4bfb-9bf3-b0f878d97318">

<img width="399" alt="correcthovershare" src="https://github.com/simularium/simularium-website/assets/24981838/f669fd46-bd3b-4176-8ee1-f33f666798df">
